src/scales/categoryScale.ts

```diff
@@ -33,23 +33,10 @@ module Plottable.Scales {
       return super.domain(values);
     }
 
-    protected _setDomain(values: string[]) {
-      super._setDomain(values);
-      this.range(this.range()); // update range
```

Comment:
As the range didn't change due to this method, I assumed this call was only in place to cause an update to `d3Scale.rangeBands` (in the `range()` method).

That same effect now occurs in `_setBackingDomain`.